sdk/storage/azure-storage-files-datalake/test/datalake_file_client_test.cpp

```diff
@@ -452,8 +452,9 @@ namespace Azure { namespace Storage { namespace Test {
       options.Metadata = RandomMetadata();
       auto res
           = fileClient.UploadFrom(fileContent.data(), static_cast<std::size_t>(fileSize), options);
+      auto lastModified = fileClient.GetProperties()->LastModified;
       EXPECT_FALSE(res->ETag.empty());
-      EXPECT_FALSE(res->LastModified.empty());
+      EXPECT_EQ(res->LastModified, lastModified);
```

Comment:
How did you test `lastModifed` is correctly parsed?

Reply:
That's actually a good question. Any proposals?

Reply:
search `IsValidDate`